src/main/java/racingcar/domain/car/CarMovementDecider.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.domain.car;
+
+import racingcar.domain.game.GameCondition;
+import racingcar.dto.Car;
+
+public class CarMovementDecider {
+    private final CarMover carMover;
+
+    public CarMovementDecider(Car car) {
+        this.carMover = new CarMover(car);
+    }
+
+    public Integer decideMovement() {
+        int movePosition;
+
+        if (GameCondition.hasDriveCondition()) {
+            movePosition = carMover.moveForward();
+        } else {
+            movePosition = carMover.stopMovement();
+        }
+
+        return movePosition;
+    }
+}
```

Comment:
이렇게 바꿔보시면 될 것 같습니다 :)
```suggestion
    public Integer decideMovement() {
        if (GameCondition.hasDriveCondition()) {
            return movePosition = carMover.moveForward();
        } 
        return  movePosition = carMover.stopMovement();
 
    }
}
```

Reply:
위와 같이 한 함수에서 return문이 여러개 존재하면 나중에 코드가 복잡해졌을때 여러지점에서 반환을 하게되어 디버깅이 어려울 수 있다고 생각했습니다! 
하지만 간결한 함수를 유지할때는 위와 같은 코드가 해결법이 될 수 있을 것 같네요! 조언 감사합니다 ㅎㅎ